### ISSUE TITLE
Feature: Add READ.ME

### ISSUE THREAD

**Issue by Layla-P** [Layla-P/HacktoberfestProject#7]
** Description: **

It would be nice to have a readme!

** Steps: **

1. It will appear on the repo in GitHub!

**Comment by CrypticEngima** [comment]
Once the Readme is done I will add a setup section for settting up and configuring the CosmosDb Emulator for the dev environment of the project.
